### PR TITLE
Added Github\HttpClient\HttpClient::$client

### PR DESCRIPTION
**Pull request by sergeyklay**
The `Github\HttpClient\HttpClient::$client` declared dynamically

### CODE REVIEW

**Comment by sergeyklay** [comment]
@cursedcoder

File: lib/Github/HttpClient/HttpClient.php [hunk 1]
```diff
@@ -20,6 +20,11 @@
  */
 class HttpClient implements HttpClientInterface
 {
+    /**
+     * @var ClientInterface
+     */
+    public $client;
```
Comment (cursedcoder) [inline lib/Github/HttpClient/HttpClient.php]: should be private variable

Comment (sergeyklay) [inline lib/Github/HttpClient/HttpClient.php]: No. Check please code

Comment (cursedcoder) [inline lib/Github/HttpClient/HttpClient.php]: what code should I check? can you please elaborate?

Comment (sergeyklay) [inline lib/Github/HttpClient/HttpClient.php]: @cursedcoder For example `lib/Github/Client.php:249` (current `master` branch):
```php
$this->getHttpClient()->client->setBaseUrl($baseUrl . '/api/v3');
```

Comment (GrahamCampbell) [inline lib/Github/HttpClient/HttpClient.php]: Making it private would be breaking anyway since it was defaulting to public already?

Comment (sergeyklay) [inline lib/Github/HttpClient/HttpClient.php]: It seems to me - yes

**Pull request merged**